subsys/mpsl/cx/thread/mpsl_cx_thread.c

```diff
@@ -207,7 +207,12 @@ static uint32_t req_grant_delay_get(void)
 
 static int32_t register_callback(mpsl_cx_cb_t cb)
 {
+	const struct device *gpiob_none = NULL;
+	struct gpio_callback *cb_none = NULL;
+	uint32_t pins_none = 0;
+
 	callback = cb;
+	gpiote_irq_handler(gpiob_none, cb_none, pins_none);
```

Comment:
This changes functionality of `mpsl_cx_register_callback()` so that this function is not working anymore according to the description in https://github.com/nrfconnect/sdk-nrfxlib/blob/2be3a9c539b86bf7496ac09202ec4df557e17fd6/mpsl/include/protocol/mpsl_cx_protocol_api.h#L201

I think the description shall be updated in the same commit this change is made.